### DISCUSSION TITLE
All pages get rotated although only one actually is

### DISCUSSION THREAD

**Discussion by jerem1e** [jsvine/pdfplumber#790] [Q&A]
I have a PDF where the second page is rotated 270 degrees and the first is not. Unfortunately I can not share the PDF as it's work related, but I think this is a more general question.
The end goal is to read the images in correct rotation so that I can read the text. 
The code I am using:
```python
import pdfplumber

pdf_path = "7113329_4009800_13008.pdf"

with pdfplumber.open(pdf_path) as pdf:
    for page in pdf.pages:
        print(page.rotation)
```
Which outputs 270 and 270, although it should put out 0 for the first page.

Maybe this the expected behavior, but it leads to having wrongly rotated images when trying to extract the from `page.images`

**Comment by jsvine** [comment]
Hmm, very interesting! Without the PDF, this will be very hard to diagnose. But thank you for opening this discussion, in case other people may have PDFs they can share that demonstrate a similar issue.

**Comment by scythe837** [comment]
maybe this helps?
i get 0 for this pdf (altough the page (text on the page) is 90 deegres rotated)

so i suppose even if the metadata of the pdf is right it might not coincide with the content on the page?
[P393964b.pdf](https://github.com/jsvine/pdfplumber/files/14884401/P393964b.pdf)

**Reply by jsvine** [reply]
Hi @scythe837, and thanks for providing that example. You ask, "so i suppose even if the metadata of the pdf is right it might not coincide with the content on the page?" That's correct, indeed. For this particular PDF, the metadata says that the the page is *not* rotated:

<img width="520" alt="Screenshot" src="https://github.com/jsvine/pdfplumber/assets/534702/507caa54-1db7-4ee6-9c46-ce62b33f813e">

(Screenshot of the PDF opened in [this tool](https://brendandahl.github.io/pdf.js.utils/browser/).)
